### ISSUE TITLE
How to get the multipath functionality working on windows?

### ISSUE THREAD

**Issue by SupradhaBhat** [private-octopus/picoquic#1674]
The multipath functionality is working in the Linux environment as shown in the screenshot attached, following are the commands used to run the client and server - 

server:
 ./picoquicdemo -c ./ca-cert.pem -k ./server-key.pem -p 4433 -w ./server_files -M 1

client:
./picoquicdemo -M 1 -A 192.168.43.1/3,192.168.43.2/4 -o .\client_files localhost 4433 server_files/index.htm

The above commands set up alternate paths for file transfer in the case of Linux
but similar results are not being shown on Windows. 

My goal is to understand why similar results are not observed while running these commands on Windows.

Linux output:
![Linux output](https://github.com/private-octopus/picoquic/assets/97398229/6370d5fe-2c18-46c0-963e-72c95be4a8fb)

Windows 11 output:
![windows_11 output](https://github.com/private-octopus/picoquic/assets/97398229/a37c79be-b409-45ea-9cdd-2b2e0c2f0c88)

**Comment by huitema** [comment]
Have you tried with a larger file? You may have a timing issue. If the handshake and the file transmission conclude, the connection will be closed before the time comes to establish alternate paths.

**Comment by SupradhaBhat** [comment]
I tried with a file of size about 7000KB but only one path is being used.

**Comment by huitema** [comment]
I just tested the running of picoquicdemo.exe on Windows. To launch the client, I used the command:
~~~
.\picoquicdemo.exe -q cclog -M 1 -A 0.0.0.0/0 -n test ::1 4433 /1000000
~~~
I do see two paths being established, but the traffic flows mostly on path 0. This is a bug in the scheduling algorithm, discussed in issue #1707. It affects both Linux and Windows versions. It should be fixed soon!

**Comment by huitema** [comment]
The option for testing multipath without an alternative address was poorly documented. It turns out that with the previous fixes, a command using the IPv6 NULL address like:
~~~
.\picoquicdemo.exe -D -q cclog -M -A ::0/0 test.privateoctopus.com 4433 /1000000
~~~
... would get the expected behavior, with load split more or less equally between the two paths. But a variation of that command using the IPv4 NULL address like:
~~~
.\picoquicdemo.exe -D -q cclog -M -A 0.0.0.0/0 test.privateoctopus.com 4433 /1000000
~~~
... would not.

I fixed the code so that the two forms of NULL address produce the same result.

Your initial trial with:
~~~
./picoquicdemo -M 1 -A 192.168.43.1/3,192.168.43.2/4 -o .\client_files localhost 4433 server_files/index.htm
~~~
... probably did not work because of configuration issues. Are you sure that the addresses 192.168.43.1/3 and 192.168.43.2/4 are valid? This could probably be debugged by looking at the result of `ipconfig` on that Windows computer.

**Comment by huitema** [comment]
There was also issue #1707, which is addressed in PR #1707.

**Comment by SupradhaBhat** [comment]
> The option for testing multipath without an alternative address was poorly documented. It turns out that with the previous fixes, a command using the IPv6 NULL address like:
> 
> ```
> .\picoquicdemo.exe -D -q cclog -M -A ::0/0 test.privateoctopus.com 4433 /1000000
> ```
> 
> ... would get the expected behavior, with load split more or less equally between the two paths. But a variation of that command using the IPv4 NULL address like:
> 
> ```
> .\picoquicdemo.exe -D -q cclog -M -A 0.0.0.0/0 test.privateoctopus.com 4433 /1000000
> ```
> 
> ... would not.
> 
> I fixed the code so that the two forms of NULL address produce the same result.
> 
> Your initial trial with:
> 
> ```
> ./picoquicdemo -M 1 -A 192.168.43.1/3,192.168.43.2/4 -o .\client_files localhost 4433 server_files/index.htm
> ```
> 
> ... probably did not work because of configuration issues. Are you sure that the addresses 192.168.43.1/3 and 192.168.43.2/4 are valid? This could probably be debugged by looking at the result of `ipconfig` on that Windows computer.

Yes the addresses are correctly taken from my ipconfig but I could not see the expected results, i.e I could see just one packet going on each of the alternate paths and the rest of the packets are going through the path[0].
